core/core_integrations/test.js

```diff
@@ -0,0 +1,78 @@
+var shim = require.once('../shim.js'),
+    cline = require('cline'),
+    chalk = require('chalk'),
+    historySize = 1024,
+    receivedCallback = null,
+    cli = null,
+    api = null,
+    hasStartedShutdown = false;
+
+exports.start = function (callback) {
+    receivedCallback = callback;
+
+    if (!exports.config.threadId) {
+        exports.config.threadId = 1;
+    }
+    if (!exports.config.senderId) {
+        exports.config.senderId = 0;
+    }
+    if (!exports.config.senderName) {
+        exports.config.senderName = 'TESTING';
+    }
+    if (!exports.config.commandHistory) {
+        exports.config.commandHistory = [];
+    }
+
+    api = api = shim.createPlatformModule({
+        commandPrefix: exports.config.commandPrefix,
+        sendMessage: function (text) {
+            console.log(chalk.bold("" + text));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2531315428/issues/source?bid=3282987&fileBranchId=3283570#l29)